.github/workflows/ci.yml

```
@@ -0,0 +1,57 @@
+name: trade-tariff/process-appendix-5a/ci
+on:
+  push:
+    branches-ignore:
+    - main
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    container:
+      image: python:3.11.2
+    steps:
+    - uses: actions/checkout@v4.1.0
+    - name: install flake8
+      run: pip install flake8==5.0.4
```

Is it important to only install this version of flake8?

---

This is technically inherited. Probably if we're going to use flake8 in the long-run for python we can just install the latest, yeah

```suggestion
      run: pip install flake8
```